client/src/client.rs

```diff
@@ -462,6 +470,30 @@ impl Client {
         self.get_config(&GetConfiguration::Value)
             .wrap_err("Failed to get configuration value")
     }
+
+    /// Gets network status seen from the peer
+    /// # Errors
+    /// Fails if sending request or decoding fails
+    pub fn get_status(&self) -> Result<serde_json::Value> {
```

Comment:
Let's de-serialize status into `Status` here and not in the tests. Client always knows the API it is working with so it is better to de-serialize into structs instead of returning raw json values.

Reply:
For this it might be better to move `Status` struct from torii into `data_model` crate.